spec.md

```diff
@@ -691,8 +697,16 @@ message VolumeContentSource {
     string id = 1;
   }
 
+  message VolumeSource {
+    // Contains identity information for the existing source volume.
+    // This field is REQUIRED.  Plugins reporting CLONE_VOLUME
```

Comment:
nit: be consistent about one or two spaces after period.

Reply:
Sorry about that.  I always leave two, but I know in this spec one is preferred, so I try to, but muscle memory sometimes gets the best of me. ;-)